src/Adapter/StubAdapter.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Queue\Adapter;
+
+use Yiisoft\Queue\Enum\JobStatus;
+use Yiisoft\Queue\Message\MessageInterface;
+
+/**
+ * Stub adapter that does nothing. Job status is always "done".
```

Comment:
It's used in tests only, so let's move it to tests too. We've got `tests/App` directory for such stubs.

Reply:
Stub extracted to separate namespace for stubs